### PR TITLE
engine: properly estimate store size

### PR DESCRIPTION
**Pull request by tabokie**
Signed-off-by: tabokie <xy.tao@outlook.com>

### What is changed and how it works?

What's Changed:

Account for snap size and raft engine size in "Store size" metrics.

### Check List 

Tests
- Unit test

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by hunterlxt — COMMENTED** [review]
Summing kv_engine and raft_engine, we can’t see each item individually

**Comment by tabokie** [comment]
@hunterlxt This PR doesn't remove any old metrics, it adds `tikv_store_size_bytes[used]`. The engine size can still be read through `tikv_engine_size_bytes`, but right now we don't have a dedicate panel for them.

File: metrics/grafana/tikv_details.json [hunk 1]
```diff
@@ -413,7 +413,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(tikv_engine_size_bytes{tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}) by (instance)",
+              "expr": "sum(tikv_store_size_bytes{tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", type=\"used\"}) by (instance)",
```
Comment (Connor1996) [inline metrics/grafana/tikv_details.json:416]: I see `tikv_engine_size_bytes` is used in a lot of spots, but you only change one place.

Comment (tabokie) [reply metrics/grafana/tikv_details.json:416]: Fixed.

**Review by hunterlxt — APPROVED** [review]
LGTM

File: metrics/grafana/tikv_details.json [hunk 2]
```diff
@@ -2908,103 +2908,6 @@
...
-          "pointradius": 5,
-          "points": false,
-          "renderer": "flot",
-          "seriesOverrides": [],
-          "spaceLength": 10,
-          "stack": true,
-          "steppedLine": false,
-          "targets": [
-            {
-              "expr": "sum(tikv_engine_size_bytes{tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}) by (instance)",
```
Comment (Connor1996) [inline metrics/grafana/tikv_details.json:2958]: why remove it

Comment (tabokie) [reply metrics/grafana/tikv_details.json:2958]: It's Store size again, duplicated.

**Review by Connor1996 — APPROVED** [review]
LGTM

**Pull request merged**